plans/libevent/plan.sh

```diff
@@ -0,0 +1,13 @@
+pkg_origin=core
+pkg_name=libevent
+pkg_version=2.0.22
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('BSD')
```

Comment:
Thanks - corrected 

> On 21 Jun 2016, at 01:45, Dave Parfitt <notifications@github.com> wrote:
> 
> In plans/libevent/plan.sh:
> 
> > @@ -0,0 +1,13 @@
> > +pkg_origin=core
> > +pkg_name=libevent
> > +pkg_version=2.0.22
> > +pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
> > +pkg_license=('BSD')
> this should be BSD-3-Clause per http://spdx.org/licenses/
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>